src/util/specific.py

```diff
@@ -60,6 +61,17 @@ def rename_custom_table(
     )
 
 
+def rename_custom_module(cr, old_module_name, new_module_name, report_details=""):
```

Comment:
I would also move this utility definition to before `rename_custom_table()` to list them sorted by magnitude (`module` > `model` ~~ `table` > `column`).

Reply:
changes done as per suggestion thanks for the review